docs/my-app-with-lazy-loaded-translations/ember-cli-build.js

```diff
@@ -28,11 +28,8 @@ module.exports = function (defaults) {
         module: {
           rules: [
             {
-              generator: {
-                filename: '[path][name]-[hash][ext][query]',
-              },
```

Comment:
I believe we can remove this when using `asset/source`.

The [documentation for `generator.filename`](https://webpack.js.org/configuration/module/#rulegeneratorfilename) said that it "works only with `asset` and `asset/resource` module types." The file names in `dist/translations` didn't show any hash.